cassandane/tiny-tests/JMAPEmail/email_get_createdmodseq

```diff
@@ -0,0 +1,106 @@
+#!perl
+use Cassandane::Tiny;
+use Time::HiRes qw( usleep gettimeofday );
+
+sub test_email_get_createdmodseq
+    :min_version_3_9 :needs_component_jmap :JMAPExtensions
+{
+    my ($self) = @_;
+    my $jmap = $self->{jmap};
+    my $imap = $self->{store}->get_client();
+    $jmap->AddUsing('https://cyrusimap.org/ns/jmap/mail');
+
+    my $cassini = Cassandane::Cassini->instance();
+    if ($cassini->bool_val('valgrind', 'enabled')) {
+        # This test is time-sensitive and valgrind is
+        # likely to violate its assumptions. We can
+        # however test that the Email/get code for
+        # createdModseq valgrinds OK.
+        xlog $self, "valgrind is enabled - running less assertive version";
+        $self->run_simplified_for_valgrind();
+        return;
+    }
+
+    xlog $self, "Append duplicate messages";
+    $mimeMessage = <<'EOF';
+From: <from@local>
+To: to@local
+Subject: test
+Date: Mon, 13 Apr 2020 15:34:03 +0200
+MIME-Version: 1.0
+Content-Type: text/plain
+
+test
+EOF
+    $mimeMessage =~ s/\r?\n/\r\n/gs;
+
+    # Sleep until the start of a second. This should result
+    # in the duplicates having the same internal date,
+    # unless the test code is running really slow.
+    my ($now_secs, $now_microsecs) = gettimeofday;
+    usleep(2 * 1000000 - $now_microsecs);
```

Comment:
Why not just set the internaldate as part of the APPEND?

Reply:
Because I was not aware that this is an option. I updated the test code.